enclave-tls/samples/enclave-tls-server/server.c

```diff
@@ -184,13 +184,12 @@ int enclave_tls_server_startup(int sockfd, enclave_tls_log_level_t log_level, ch
 		}
 
 		close(connd);
+		enclave_tls_cleanup(handle);
 	}
 
 	return 0;
 
 err:
-	/* Ignore the error code of cleanup in order to return the prepositional error */
-	enclave_tls_cleanup(handle);
 	return -1;
```

Comment:
why remove enclave_tls_cleanup when err happens?

Reply:
`handle` has moved to while scope already.

Reply:
I mean during session once something is wrong, the enclave_tls_clean will be invoked, goto err should use enclave_tls_clean to release some memory

Reply:
This should be done in real scenarios, but if an error occurs in the sample, it means exiting. In order to simplify the implementation, let the process exit to clean up resources.